cmd/rac/output.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+func info() {
+	println(`
+### CSVs ###
+
+The first row:
+- "CODE": Means the following column says what version of the code produced
+  the output.
+- "v0.2.2": The code version
+- "RAMES": Means the following column says what RAMSES specification was used.
+- "SPU045-S2:6F": The RAMSES version
+- "INNOSAT": Means the following column says what INNOSAT specification was
+  used.
+- "IS-OSE-ICD-0005:1": The INNOSAT version
+- "AEZ": Means the following column says what AEZ specification was used.
+- "AEZICD002:H": The AEZ version
+
+The header row starts with a couple of columns common to all output and then
+follows columns specific to each file
+
+
+- File
+  The full path to the rac-file on the computer that produced the csv
+- ProcessingDate
+  The local time when the file was processed
+- RamsesTime (Ramses Header)
+  The time when the ramses file was created (UTC)
+
+- QualityIndicator (Ramses TM Header)
+  Indicates whether the transported data is complete or partial
+  (0 = Complete, 1 = partial).
+- LossFlag (Ramses TM Header)
+  Used to indicate that a sequence discontinuity has been detected
+- VCFrameCounter (Ramses TM Header)
+  Counter of the transfer frame the payload packet arrived in.
+
+- SPSequenceCount (Innosat Source Header)
+- TMHeaderTime (Innosat TM Header)
+  The time of the TM packet creation (UTC)
+- TMHeaderNanoseconds
+  The time of the TM packet creation (nanoseconds since epoch)
+- SID
+  The name of the SID or empty if the packet has no SID
+- RID
+  The name of the RID or empty if the packet has no RID
+
+Note that each line should have a SID or a RID depending on packet type
+
+All files also have a final common column
+
+- Error
+  If an error occurred it will be written here.

```

Comment:
Can be a list of errors, right?

Reply:
No the DataRecord object only holds one error so I don't see how that would happen.

Reply:
Right, yes. CCD only. Will you make an issue for the remainder?

Reply:
The error is for any packet, we are confuse